arangod/Aql/AqlValue.cpp

```diff
@@ -624,21 +580,19 @@ double AqlValue::toDouble(bool& failed) const {
   failed = false;
   auto t = type();
   switch (t) {
-    case VPACK_INLINE_INT48:
-      return static_cast<double>(_data.shortNumberMeta.data.int48.val);
     case VPACK_INLINE_INT64:
       // TODO(MBkkt) Not all int64_t fit to double without precision lost
-      return static_cast<double>(
-          basics::littleToHost(_data.longNumberMeta.data.intLittleEndian.val));
+      return static_cast<double>(absl::little_endian::ToHost(
```

Comment:
I think it makes sense to add an explicit `#include` directive for the absl header that we require in this file now.
And we can now remove the include header for the endianness functionality in `Basics/Endian.h`.

Reply:
#include <absl/strings/numbers.h> for endian

Basics/Endian removed.